config.json

```diff
@@ -3,7 +3,7 @@
   "slug": "haskell",
   "active": true,
   "status": {
-    "concept_exercises": false,
+    "concept_exercises": true,
```

Comment:
I was unable to determine from the official documentation in https://github.com/exercism/docs/blob/main/building/tracks/config-json.md whether it's acceptable to change this to true when all concept exercises are still either status `beta` or `wip`. If I do not get an answer from a knowing member by the time it comes to merge this PR, I will be removing this change from this PR and it will need to be resubmitted at a later time.

Reply:
This should not be enabled unless we're ready to enable learning mode.

Reply:
Changed concept_exercises back to false.